.storybook/main.js

```diff
@@ -60,9 +62,25 @@ module.exports = {
       },
     };
 
+    storybookConfig.module.rules.push({
+      resourceQuery: /raw/,
+      type: 'asset/source',
+    });
+
     storybookConfig.module.rules.push({
       test: /\.(ts|tsx)$/,
-      loader: require.resolve('babel-loader'),
+      use: [
+        {
+          loader: require.resolve('babel-loader'),
+        },
+        {
+          loader: require.resolve('ts-loader'),
```

Comment:
> I couldn't get fork-ts-checker-webpack-plugin to work with storybook - that would type check in another thread and speed up storybook. I think the initial slowdown isn't too much, but if it is I can set ts-loader to transpile only which should speed it up, but not type check.

Yeah the startup of the storybook is way to long IMHO. Making sure that my changes don't break stories is already something that felt like a chore so if doing that will mean spending a minute just waiting it'll make people even less likely to do it.

At the moment I couldn't even get the story book to run because of type errors.

I guess we could turn off type checking there until we figure out how to make it faster?

Reply:
Sure thing, I'll turn it off!